### PR TITLE
news: add news for v2.0.0-beta

### PR DESCRIPTION
**Pull request by ajeddeloh**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+
+  Features:
+    - Add configuration spec 3.1.0-experimental
+    - Add support for proxies in spec 3.1.0-experimental
+    - Validate hard links do not link to directories
+    - Validate paths do not include links specified in the config
+
+  Bug Fixes:
+    - Include major version in go.mod correctly
+    - Fix systemd unit SELinux relabeling
```
Comment (bgilbert) [inline NEWS]: `Fix SELinux relabeling of systemd unit files`?

File: NEWS [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+    - Update documentation to be correct for spec 3.0.0+
```
Comment (bgilbert) [inline NEWS]: Maybe s/to be correct //?

File: NEWS [hunk 3]
```diff
@@ -1,3 +1,25 @@
+26-Apr-2019 Ignition 2.0.0-beta
+
+  Features:
+    - Add configuration spec 3.1.0-experimental
+    - Add support for proxies in spec 3.1.0-experimental
```
Comment (bgilbert) [inline NEWS]: Maybe "Allow specifying network proxies in..."?

File: NEWS [hunk 4]
```diff
@@ -1,3 +1,25 @@
...
+  Bug Fixes:
+    - Include major version in go.mod correctly
+    - Fix systemd unit SELinux relabeling
+    - Update documentation to be correct for spec 3.0.0+
+
+  Changes:
+    - Remove all deprecated fields in configuration specs
+    - Remove `ec2` platform id in favor of `aws`
+    - Remove `pxe` platform as it is not a platform
+    - Ignition now fails if files, links, and directories conflict after
```
Comment (bgilbert) [inline NEWS]: Maybe s/Ignition now fails/Fail/?

File: NEWS (hunk 3 above)
Comment (ajeddeloh) [reply NEWS]: How about "Allow specifying HTTP(S) proxies in...."

Comment (bgilbert) [reply NEWS]: Sure.

**Comment by ajeddeloh** [comment]
Pushed fixes.

**Pull request merged**